bdai_ros2_wrappers/bdai_ros2_wrappers/executors.py

```diff
@@ -665,8 +678,9 @@ def shutdown(self, timeout_sec: typing.Optional[float] = None) -> bool:
             with self._spin_lock:
                 # rclpy.executors.Executor base implementation leaves tasks
                 # unawaited upon shutdown. Do the housekeepng.
-                for task, entity, _ in self._tasks:
-                    task = AutoScalingMultiThreadedExecutor.Task(task, entity)
+                for task, entity, node in self._tasks:
+                    Task = AutoScalingMultiThreadedExecutor.Task
+                    task = Task(task, entity, node)
```

Comment:
Is this about the line length? (I also see the addition of node as a parameter)

Reply:
I'm OCD'ing. No need to decouple the two lines. See 3159cdcf2848645850c50bcb3e94df13765631a0